### ISSUE TITLE
Any way to eventually help and get this tool in English too?

### ISSUE THREAD

**Issue by muscularcandy67** [CMHopeSunshine/LittlePaimon#282]
The question is the same as title, I am in no way able to speak nor read chinese, and from a quick look on this seems like this is a nice tool to have, and, with a little help from the AI translator, I want to eventually port this in English too for the global audience.

**Comment by kongxiangyi88** [comment]
The following content is machine translation. Because of different chat software, this may be very difficult. The translation of the project ontology should not be too difficult. You can wait for the author's reply.

**Comment by YvetteVv** [comment]
It seems to require huge work to support dual language.

**Comment by ljx-code** [comment]
Most of the functions of this project are based on the app called Mi You She. I'm not sure whether it can be used in non Chinese regions.

**Comment by Moshimoxi** [comment]
> Most of the functions of this project are based on the app called Mi You She. I'm not sure whether it can be used in non Chinese regions.

I remember that Miyushe(bbs.mihoyo.com) could not be visited outside of China.

**Comment by nicklly** [comment]
This is a good idea, but don't forget the most important point, LittlePaimon is use Mi You She （bbs.mihoyo.com）API. If you try to use it for other server players, it will cause some problems

**Comment by ljx-code** [comment]
> > Most of the functions of this project are based on the app called Mi You She. I'm not sure whether it can be used in non Chinese regions.
> 
> I remember that Miyushe(bbs.mihoyo.com) could not be visited outside of China.
That's true, but the api of the whole project needs to be replaced, and the third-party websites that rely on need to be replaced in non Chinese regions, which is no less than the workload of project reconstruction.

**Comment by 3944Realms** [comment]
I am support this idea . 

> Most of the functions of this project are based on the app called Mi You She. I'm not sure whether it can be used in non Chinese regions.
